index.js

```diff
@@ -33,30 +56,12 @@ try {
   await page.keyboard.type(selectors.password);
   await page.click(selectors.login_button);
   await page.waitFor(2000);
+  if (page.url() == selectors.sec_key_tfa_url) {
+    await page.goto(selectors.otp_tfa_url);
+    await handle_otp(page);
+  }
   if (page.url() == selectors.otp_url) {
```

Comment:
At line 63, ... to add check if the page url was OTP TFA in addition to plain OTP. Something like this

```suggestion
  if (page.url() == selectors.otp_url || page.url() == selectors.otp_tfa_url) {
```